pkg/kubelet/cm/pod_container_manager_stub.go

```diff
@@ -19,9 +19,57 @@ package cm
 import (
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"sync"
 )
 
+type CGroupPods struct {
```

Comment:
I'm confused as to where this stub code gets called and tested on Linux. Looks like the stub is only instantiated on `unsupported` platforms and Windows.

Reply:
This was introduced to facilitate TestDeleteOrphanedMirrorPods in pkg/kubelet/kubelet_test.go
Please note that Kubelet#cleanupOrphanedPodCgroups uses a goroutine to destroy the orphan cgroup.
That is why some synchronization is needed in test code to avoid Data Race.
The formation you see now is the result of resolving the Data Races.

Also note: no production code is altered for TestDeleteOrphanedMirrorPods.

Reply:
Thanks... That helps.